roles/ci_gen_kustomize_values/tasks/generate_snippets.yml

```diff
@@ -77,11 +86,22 @@
       {{
         _raw_manifests | reject('equalto', _config_map_content)
       }}
+    _cifmw_gen_kustomize_values_reject_expressions: >-
+      {% set _final_list = []                                                  -%}
+      {% for entry in (_cifmw_gen_kustomize_values_reject_extra_expressions +
+                      cifmw_ci_gen_kustomize_values_remove_keys_expressions)   -%}
+      {%   if entry is string                                                  -%}
+      {%     set _ = _final_list.append(entry)                                 -%}
+      {%   else                                                                -%}
+      {%     set _ = _final_list.extend(entry)                                 -%}
+      {%   endif                                                               -%}
+      {% endfor                                                                -%}
+      {{ _final_list | unique }}
```

Comment:
I think we can just concatenate the lists, and then `| flatten | unique`?
```
{{ 
  (_cifmw_gen_kustomize_values_reject_extra_expressions +
   cifmw_ci_gen_kustomize_values_remove_keys_expressions)
  | flatten | unique  
}}
```

Reply:
@hjensas the approach I took allows the user to pass individual regexes, thus the if/else

Reply:
@hjensas Good catch man, i didn't thought about using `flatten` this time.